index.d.ts

```diff
@@ -80,7 +80,7 @@ declare namespace tuiChart {
     interface IThemeConfig {
         chart?: {
             fontFamily?: string;
-            background?: string;
+            background?: IThemeRange;
```

Comment:
As the naming of the `IThemeRange` type is used universally, it seems necessary to change the name.

How about a naming of `IThemeBackgroundConfig`?

Reply:
@dana-c0914 I'll Apply it after merge.

Reply:
@jung-han thank you